pkg/intoto/os.go

```diff
@@ -52,11 +52,11 @@ func verifyOnOS(verificationDir string) error {
 		return fmt.Errorf("cannot read root layout pubkeys in %v: %v", verificationDir, err)
 	}
 	for _, filename := range filenames {
-		err = rootLayoutPubKey.LoadPublicKey(filename)
+		err = rootLayoutPubKey.LoadKey(filename, "rsassa-pss-sha256", []string{"sha256", "sha512"})
```

Comment:
Any thoughts on this? I think it's fine for now and when we get #80 approved, we can revisit?